bidsify/bidsify.py

```diff
@@ -118,7 +133,7 @@ def bidsify_workflow(dicom_dir, heuristics, subject, session=None, output_dir='.
     if participants_file.is_file():
         participant_df = pd.read_table(
             participants_file, index_col='participant_id')
-        data = manage_dicom_dir(dicom_dir)
+        data = manage_dicom_dir(dicomdir)
```

Comment:
Might as well change `manage_dicom_dir` to `manage_dicomdir` for internal consistency

Reply:
done